pages/apple-silicon/how-to/setup-vm-with-utm.mdx

```diff
@@ -0,0 +1,80 @@
+UTM is free and open-source software that leverages Apple's Hypervisor framework and QEMU to provide virtualization capabilities.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An [SSH key](/organizations-and-projects/how-to/create-ssh-key/)
+- A [Mac mini](/apple-silicon/how-to/create-mac-mini/)
+- [VNC access](/apple-silicon/how-to/access-remote-desktop-mac-mini/) to your Mac
+
+## Installating UTM
+
+1.  Download and install UTM on the Mac mini:
+    *   From the official [Mac App Store](https://apps.apple.com/us/app/utm-virtual-machines/id1538878817) or
+    *   Directly from the [official website](https://getutm.app/).
+2.  Download the macOS IPSW file.
+    The IPSW file is a compressed a file format used to install iOS, iPadOS, tvOS, HomePod, watchOS, macOS firmware for devices equipped with Apple silicon.
+    *   Obtain the macOS IPSW file from the official [Apple Developer website](https://developer.apple.com/download/).
+    <Message type="note">
+      You need an [Apple Developer account](/apple-silicon/troubleshooting/cant-create-apple-account/) to access these files.
+    </Message>
+3.  Create a new virtual machine:
+    *   Open UTM and click the "+" button to create a new VM.
+    *   Select "Virtualize" and then "macOS".
+    *   Browse and select your downloaded IPSW file.
+    *   Configure your VM's properties:
+        *   Set your desired CPU cores (recommended at least 2 cores).
+        *   Allocate RAM (recommended at least 4GB).
+            <Lightbox src="scaleway-aps-utm-1.webp" size="medium" alt="The RAM allocation window of UTM with 4096 MB of RAM selected" />
+        *   Configure the disk storage size (recommended at least 64GB).
+            <Lightbox src="scaleway-aps-utm-2.webp" size="medium" alt="The disk size confgiration window of UTM with 64GB of storage chosen" />
+        *   Click "Save" to create the VM.
+        <Lightbox src="scaleway-aps-utm-3.webp" size="medium" alt="Summary of the VM configuration displaying the Save button to apply the configuration" />
+4.  Start and set Up your macOS VM:
+    *   Once the VM is installed with your properties, click the "Run" button in UTM.
+    *   A GUI window will open displaying your VM.
+        <Lightbox src="scaleway-aps-utm-4.webp" size="medium" alt="Mac OS setup wizard" />
+    *   Follow the macOS installation process:
+        *   Select language preferences.
+        *   Set up your Apple ID (optional).
+        *   Create a user account with username and password.
+        *   Choose time zone and other basic settings.
+        *   Wait for the installation to complete.
+
+## Troubleshooting
```

Comment:
shouldn't this section be its own troubleshooting page?

Reply:
Not sure if it is worth an own page as it is very specific linked to this guide, 99% of users might never encounter this issue